packages/php/src/Metrics.php

```diff
@@ -106,11 +82,10 @@ public function __construct(string $api_key, string $group_handler, array $optio
             'timeout' => $curl_timeout,
         ]);
 
-        /** @psalm-suppress DeprecatedClass */
-        $this->package_version = Versions::getVersion(self::PACKAGE_NAME);
+        $this->package_version = InstalledVersions::getVersion(self::PACKAGE_NAME);
         $this->cache_dir = Factory::createConfig()->get('cache-dir');
 
-        $this->user_agent = 'readme-metrics-php/' . $this->package_version;
+        $this->user_agent = 'readme-metrics-php/' . $this->package_version ?? 'unknown';
```

Comment:
`InstalledVersions::getVersion()` may return `null` if it can't find what version we're running so just in case we're going to have the version be `unknown` in the user agent instead of nothing.